src/pysource/adjoint_test_F.py

```diff
@@ -10,6 +10,8 @@
 parser = ArgumentParser(description="Adjoint test args")
 parser.add_argument("--tti", default=False, action='store_true',
                     help="Test acoustic or tti")
+parser.add_argument("--viscoacoustic", default=False, action='store_true',
```

Comment:
FYI these are debugging script that aren't run by the CI but thanks fr updating them